lib/Activity.py

```diff
@@ -163,14 +165,17 @@ def __str__(self):
                 end = start
 
             duration = self.get_duration(start, end)
-            table.add_row([
-                start,
-                state['state'],
-                state['workflow_id'],
-                state['status'],
-                duration,
-                state['comment']
-            ])
+            try:
```

Comment:
Rather than adding a try/except and skipping the bad entries, we should:
  -- Check for 'sessionid' while reading the dict in `load_activity_dict`
 -- In the `__str__` function, check the state keys against ` ACTIVITY_NEW_STATE` and initialize them to the default values if they don't exist.